products/generated/response_v1/summary_full_invoice_v4.rst

```diff
@@ -0,0 +1,246 @@
+########
+Document
+########
+:Mindee ID: 656c2ec1-0920-4556-9bc2-772162bc698a
+:Filename: invoice.pdf
+
+Inference
+#########
+:Product: mindee/invoices v4.2
+:Rotation applied: Yes
+
+Prediction
+==========
+:customer_address: 1954 Bloon Street West Toronto, ON, M6P 3K9 Canada
+:customer_company_registrations:
+:customer_name: JIRO DOI
+:date: 2020-02-17
+:document_type: INVOICE
+:due_date: 2020-02-17
+:invoice_number: 0042004801351
+:line_items:
+  * :description: S)BOIE 5X500 FEUILLES A4
+    :polygon: Polygon with 4 points.
+    :product_code:
+    :quantity:
+    :tax_amount:
+    :tax_rate:
+    :total_amount: 2.63
+    :unit_price:
+  * :description: RAM 500F DCP BLANC A4 100G
+    :polygon: Polygon with 4 points.
+    :product_code:
+    :quantity:
+    :tax_amount:
+    :tax_rate:
+    :total_amount: 0.98
+    :unit_price:
+  * :description: PQ 960 ETIQUETTES L METAL
+    :polygon: Polygon with 4 points.
+    :product_code:
+    :quantity:
+    :tax_amount:
+    :tax_rate:
+    :total_amount: 4.07
+    :unit_price:
+  * :description: CARTOUCHE L NR BROTHER TN247BK
+    :polygon: Polygon with 4 points.
+    :product_code:
+    :quantity:
+    :tax_amount:
+    :tax_rate:
+    :total_amount: 9.47
+    :unit_price:
+  * :description: PQ20 ETIQ ULTRA RESIS METAXXDC
+    :polygon: Polygon with 4 points.
+    :product_code:
+    :quantity:
+    :tax_amount:
+    :tax_rate:
+    :total_amount: 4.31
+    :unit_price:
+  * :description: FO2 IMPRIM MULTIFONCT MFC-L3770CDW
+    :polygon: Polygon with 4 points.
+    :product_code:
+    :quantity:
+    :tax_amount:
+    :tax_rate:
+    :total_amount: 120.0
+    :unit_price:
+  * :description: Platinum web hosting package Down 35mb, Up 100mb
+    :polygon: Polygon with 4 points.
+    :product_code:
+    :quantity: 1.0
+    :tax_amount:
+    :tax_rate:
+    :total_amount: 65.0
+    :unit_price: 65.0
+  * :description: 2 page website design Includes basic wireframes, and responsive templates
+    :polygon: Polygon with 4 points.
+    :product_code:
+    :quantity: 3.0
+    :tax_amount:
+    :tax_rate:
+    :total_amount: 2100.0
+    :unit_price: 2100.0
+  * :description: Mobile designs Includes responsive navigation
+    :polygon: Polygon with 4 points.
+    :product_code:
+    :quantity: 1.0
+    :tax_amount:
+    :tax_rate:
+    :total_amount: 250.0
+    :unit_price: 250.0
+:locale:
+  * :currency: EUR
+    :language: fr
```

Comment:
this is not a list

Reply:
Yep, for some reason `orientation` shows up as one as well :thinking:, probably messed up something on my end